index.html

```diff
@@ -64,6 +64,34 @@
   <div class="well">
   <h2 id="rewrap">Change container (rewrap)</h2>
 
+  <!-- Basic rewrap command -->
+  <span data-toggle="modal" data-target="#basic-rewrap"><button type="button" class="btn btn-default" data-toggle="tooltip" data-placement="bottom" title="Basic rewrap command">Basic rewrap command</button></span>
+  <div id="basic-rewrap" class="modal fade" tabindex="-1" role="dialog">
+    <div class="modal-dialog modal-lg">
+      <div class="modal-content">
+        <div class="well">
+          <h3>Rewrap a file</h3>
+          <p><code>ffmpeg -i <i>input_file.ext</i> -c copy -map 0 <i>output_file.ext</i></code></p>
+          <p>This script will rewrap a video file. It will create a new video video file where the inner content (the video, audio, and subtitle data) of the original file is unchanged, but these streams are rehoused within a different container format.</p>
```

Comment:
I like the  beginner-friendly terminology here - it's probably for the best to not talk about streams/remuxing etc if the target audience for this recipe is beginners.

Reply:
Do you think it's worth adding a note on terminology (that rewrapping is also called remuxing)?

Reply:
I think so actually -I've found that any of the muxing terms can kinda scare beginners off a bit as it sounds a lot more technical than it is. Not sure if saying 'remuxing' is prefereable to 're-multiplexing'? Maybe both - 'Rewrapping is also called re-multiplexing/remuxing' ?

Reply:
I've added a new commit which notes this. I take your point, but I do think it's good to mention the different names, for demystification! And so people aren't confused about whether remuxing means something different.

Reply:
Looks great!

Reply:
It’s a good thing to mention both muxing and wrapping, because people coming from other languages than English sometimes find the one and sometimes the other more… natural.